hw/ip/otbn/dv/otbnsim/sim/dmem.py

```diff
@@ -139,7 +139,7 @@ def dump_le_words(self) -> bytes:
 
     def load_u256(self, addr: int) -> int:
         '''Read a u256 little-endian value from an aligned address'''
-        assert 0 == addr % 32
+        assert 0 == addr % 32, "Unable to read 256b word from unaligned address 0x%x." % addr
```

Comment:
This isn't supposed to be triggered by bad OTBN code: if we hit this, we've found a bug in the ISS code that calls it. Can you trace this back up the call stack and fix it there?

Reply:
I hit this problem some time deep in a debugging session, I can't reproduce it any more unfortunately. I dropped the patch for now, let's revisit it when the problem reappears.